crates/rpc-client/src/pathfinder/client.rs

```diff
@@ -1,10 +1,19 @@
+use reqwest::{Response, StatusCode};
 use serde::de::DeserializeOwned;
 use serde::Deserialize;
 use serde_json::json;
 use starknet_types_core::felt::Felt;
 
 use crate::pathfinder::proofs::{PathfinderClassProof, PathfinderProof};
 
+#[derive(Debug, thiserror::Error)]
+pub enum ClientError {
+    #[error("Encountered a request error: {0}")]
+    ReqwestError(#[from] reqwest::Error),
```

Comment:
From what I've seen of Rust errors, shouldn't this just be `Reqwest`? The enum is already named "Error".

Reply:
Wow, definitely don't like that! But at least you've made your variants consistent